assignments/arrays.js

```diff
@@ -65,32 +65,52 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
 console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
-
-
+inventory.forEach(obj => {
+    if (obj.id === 33) {
+      console.log(
+        `Car ${obj.id} is a ${obj.car_year} ${obj.car_make}  ${obj.car_model}`
+      );
+    }
+  });
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[inventory.length - 1];
+console.log(`The make is ${lastCar.car_make} and the model is ${lastCar.car_model}`);
 
 // ==== Challenge 3 ====
-// The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+// The marketing team wants the car models listed alphabetically on the website. 
+//Sort all the car model names into alphabetical order and log the results in the console
+let carModels = []; 
+for(let i = 0; i < inventory.length; i++)
+{carModels.push(inventory[i].car_model)}
```

Comment:
Formatting is a bit off here. Consider using your VS code extensions to clean it up.